### ISSUE TITLE
Doesnt work with google cloud shell

### ISSUE THREAD

**Issue by dillythecoder** [mxrch/GHunt#289]
Error when ran on google cloud shell:
      platform: Linux-5.10.68+-x86_64-with-debian-10.11
        pip: 18.1
        setuptools: 58.5.3
        setuptools_rust: 0.12.1
        =============================DEBUG ASSISTANCE=============================

    error: can't find Rust compiler

    If you are using an outdated pip version, it is possible a prebuilt wheel is available for this package but pip is not able to install from it. Installing from the wheel would avoid the need for a Rust compiler.

    To update pip, run:

        pip install --upgrade pip

    and then retry package installation.

    If you did intend to build this package from source, try installing a Rust compiler from your system package manager and ensure it is on the PATH during installation. Alternatively, rustup (available at https://rustup.rs) is the recommended way to download and update the Rust compiler toolchain.

    This package requires Rust >=1.41.0.

    ----------------------------------------
Command "/usr/bin/python3 -u -c "import setuptools, tokenize;__file__='/tmp/pip-install-j6qihk5l/cryptography/setup.py';f=getattr(tokenize, 'open', open)(__file__);code=f.read().replace('\r\n', '\n');f.close();exec(compile(code, __file__, 'exec'))" install --record /tmp/pip-record-2meh8vim/install-record.txt --single-version-externally-managed --compile --user --prefix=" failed with error code 1 in /tmp/pip-install-j6qihk5l/cryptography/

**Comment by RosyHorus** [comment]
I'm not an expert or anything.. I just did those commands and then ran  `python3 -m pip install -r requirements.tx` again to test if all requirements been satisfied.. any ways, I found that it's common problem when trying to install "cryptography".. it requires "rust" so anyways.. the following commands would solve the problem by installing rust, and scipy then run the requirements.txt to test if all are satisfied :D 

1. `curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh`
2. `pip3 install --upgrade setuptools`
3. `pip3 install --upgrade pip`
4. `python3 -m pip install --user numpy scipy matplotlib ipython jupyter pandas sympy nose`
5. `pip install cryptography`
6. `python3 -m pip install -r requirements.tx`
